.env.example

```
@@ -166,4 +166,7 @@ VAPID_PRIVATE_KEY=
 ## Currency Conversion
 CURRENCY_API_KEY=your_key_here
 CURRENCY_BASE=GBP
-CURRENCY_CACHE_TTL=24
\ No newline at end of file
+CURRENCY_CACHE_TTL=24
+
+## Flint Assistant
+ASSISTANT_MAX_EVENTS=200
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)